docs/tutorials/search-v2_1.md

```diff
@@ -0,0 +1,34 @@
+# Amundsen +
+
+The goal of this tutorial is to explain what the new search functionality offers, how to transition into using `v2/search` search service endpoint out of the box as well as how to customize it ahead of deprecation of the old search endpoints. The updated search service offers [fuzziness](https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#fuzziness), word [stemming](https://www.elastic.co/guide/en/elasticsearch/reference/8.1/stemming.html), and [boosted search ranking](https://www.elastic.co/guide/en/elasticsearch/reference/8.1/query-dsl-rank-feature-query.html) based on usage.
```

Comment:
Nit:
I still think it's worth it to explain what is `v2/search` API (searches all resources and handles filtered and unfiltered search) and how it related to the proxies. 

I.e. there's a really old proxy that can't support v2/search API at all. 
There's a new proxy that supports v2/search API but doesn't have any of the search quality enhancements, since those need new mappings to be created. 
And finally there's the latest proxy, supports v2/search API and has the search quality improvements that rely on the new mappings being created in databuilder. It will fall back to the old V2 proxy if it cannot find the new mappings.